### ISSUE TITLE
Pagination Example

### ISSUE THREAD

**Issue by fallenpeace** [MaksTr/meteor-mantra-kickstarter#32]
In the read me it says there is pagination, so I'm looking through the codebase to find it, but i can seem to find it, can someone point me in the right direction?

**Comment by gantu** [comment]
I also searched for this but without luck.  I think it is not developed yet.

**Comment by sandervanhooft** [comment]
You are right that pagination is mentioned in the Readme, _but under the **Roadmap** section._ So it is yet to be developed.

If you have found some reference material or came up with a solution, please share! :)

**Comment by makstr** [comment]
Here is a good medium text on the subject

https://medium.com/apollo-stack/understanding-pagination-rest-graphql-and-relay-b10f835549e7#.krvgs9hi9 https://medium.com/apollo-stack/understanding-pagination-rest-graphql-and-relay-b10f835549e7#.krvgs9hi9
